### ISSUE TITLE
Is this project deprecated in favor of tdlib-purple?

### ISSUE THREAD

**Issue by nodiscc** [majn/telegram-purple#554]
I just realized by reading the current issues; that this plugin is more or less deprecated, and that a natural successor would be https://github.com/ars3niy/tdlib-purple. Is that right? I have only been using telegram-purple for some weeks, had the same issues as decribed in the bug tracker (mostly problems with encrypted chats), and tdlib supposedly fixes all of these.

There is even a [fixed-by-tdlib](https://github.com/majn/telegram-purple/issues?q=is%3Aissue+is%3Aopen+label%3Afixed-by-tdlib) issue label which seems to indicate that maintainers acknowledge this...

If this is the case, @majn @BenWiederhake what do you think about adding a **notice** on top of the README?

> **This project is currently halted, see https://github.com/ars3niy/tdlib-purple for a possible replacement**

**Comment by BenWiederhake** [comment]
Sigh. Yes, you're right, but I don't like it.

You see, tdlib-purple still is lacking some features, most prominently [secret chats](https://github.com/ars3niy/tdlib-purple/issues/11). (Huh, there seem to be no big missing features other than this.)

I'll add a notice, along with a fresh grab of the translations and a new, perhaps final release.

Fun fact: I created the "fixed-by-tdlib" label long before tdlib-purple existed. Back in the days I thought we would switch this project to tdlib.

**Comment by nodiscc** [comment]
I was just browsing issues in the tdlib-purple repository, I see your point, tdlib is incomplete because it currently lacks secret chats.

Secret chats are buggy in telegram-purple for me anyway so I'll be switching to tdlib and see how it goes.

I still see a value in adding a notice to the readme (maybe rephrase it to `(work in progress) for a possible replacement`), it will drive users and possible contributors to the new project. I was beginning to lose hope when I saw bug reports piling up here, until I found tdlib-purple - it looks promising. Thank you for maintaining it
